src/core/Browser.js

```diff
@@ -20,7 +20,7 @@ const chrome = userAgentContains('chrome');
 const safari = !chrome && userAgentContains('safari');
 
 // @property mobile: Boolean; `true` for all browsers running in a mobile device.
-const mobile = typeof orientation !== 'undefined' || userAgentContains('mobile');
+const mobile = typeof Screen.orientation !== 'undefined' || userAgentContains('mobile');

```

Comment:
```suggestion
const mobile = typeof screen.orientation !== 'undefined' || userAgentContains('mobile');

```

Don't just copy paste the first thing that looks like it fits ... and it can be expected that you test before you create a PR.

1.  it is `screen.orientation` and not `Screen.orientation`
2. `screen.orientation` is also on desktop browsers supportet, `orientation` is only on mobile browsers available -> `typeof screen.orientation !== 'undefined'` will be always true

https://developer.mozilla.org/en-US/docs/Web/API/Screen/orientation
https://developer.mozilla.org/en-US/docs/Web/API/Window/orientation